### ISSUE TITLE
Names of snapshots in Celluloid and in K-Lite Codec Pack

### ISSUE THREAD

**Issue by Ubuntu-2020** [celluloid-player/celluloid#572]
We will talk about the file names that media players gives if you take a snapshots.
I can take a snapshot in the Celluloid media player.
Let's say we watch the movie **Batman.avi**.
If I take a snapshots in **Celluloid** while watching a movie they will have the following names:
- celluloid-shot0001.jpg
- celluloid-shot0002.jpg

It is very good that the **Celluloid media player** can take pictures from movies but I believe that Celluloid does not give them optimal names! Pay attention to the [**K-Lite Codec Pack**](https://en.wikipedia.org/wiki/K-Lite_Codec_Pack).

If I take a snapshots in **K-Lite Codec Pack** while watching a movie they will have the following names: 
- Batman.avi_snapshot_00.04.13_[2020.07.26_12.37.45].jpg
- Batman.avi_snapshot_00.05.42_[2020.07.26_12.40.12].jpg

That is, first of all, **K-Lite Codec Pack** automatically gives **the same name as the video file** itself!

Second, the **K-Lite Codec Pack** automatically specifies the **time from the video file when the snapshot was taken**!

It is very good.

In other words, I would very much like **Celluloid** to give the names of snapshots the same as the **K-Lite Codec Pack** gives them!

PS: I would very much like to send this request to the **VLC media player** as well but they are not allowed to leave issue on this site :-( So wouldn't it be a violation if I mentioned some **VLC developers** here? Maybe they'll like my idea and move it to their own place?

**Comment by gnome-mpv** [comment]
You can already configure the filename using the mpv option [`screnshot-template`](https://mpv.io/manual/master/#options-screenshot-template). To get filenames like your example, put this in your mpv config file:

```
screenshot-template=%f_snapshot_%wH.%wM.%wS_[%tY.%tm.%td_%tH.%tM.%tS]
```

I think it's too long to use as the default template though. I'm thinking of using something like this instead: `celluloid-Batman.avi-00:42:13.123.jpg`.

> PS: I would very much like to send this request to the VLC media player as well but they are not allowed to leave issue on this site :-( So wouldn't it be a violation if I mentioned some VLC developers here? Maybe they'll like my idea and move it to their own place?

Feel free to mention them, but why not just open an issue on [their bug tracker](https://trac.videolan.org/vlc/) instead?

**Comment by Ubuntu-2020** [comment]
@gnome-mpv Thank you very much!!!

@gnome-mpv _I'm thinking of using something like this instead: `celluloid-Batman.avi-00:42:13.123.jpg`._
I agree with you!
But I would remove the word _**celluloid**_: `Batman.avi-00:42:13.123.jpg`
If possible, add this default template to the media player. Or you can create a survey of users **Celluloid media player** about which screenshot names they prefer.

@gnome-mpv _why not just open an issue on their bug tracker instead?_
Thank you very much I didn't know about this site.

@gnome-mpv _Feel free to mention them._
Thank you for this opportunity.
Dear developers of the **VLC media player** (@chouquette @funman @cmassiot @carolanitz @mfkl @thresheek @Mikanbu), please pay attention to this issue. I really hope that in future releases of the **VLC media player** you will be able to implement this innovation. But judging by this similar **[topic](https://trac.videolan.org/vlc/ticket/19118)** which has been open for three years so far nothing has changed.

**Comment by gnome-mpv** [comment]
>  But I would remove the word celluloid: Batman.avi-00:42:13.123.jpg
If possible, add this default template to the media player.

I think having `celluloid` in there will reduce the chance that the resulting filename will conflict with existing files, but I guess the chance of that happening is pretty low.

> Or you can create a survey of users Celluloid media player about which screenshot names they prefer.

#573

**Comment by stephan-dev** [comment]
TL;DR - Didn't work for me. I've had to add double quotes `screenshot-template="%f_snapshot_%wH.%wM.%wS_[%tY.%tm.%td_%tH.%tM.%tS]"`  

Grand beginner, my first mpv configuration file. No need for headers, includes or anything. But for me, double quotes made it work, cf. https://mpv.io/manual/stable/#configuration-files  
Current Celluloid version for my linux OS is `0.17`, which is far behind I know.
